README.md

```diff
@@ -98,7 +98,7 @@ However, there are some instances where you wish to override this, e.g.
 To help with this, there are two mechanisms for overriding the tinymce.min.js URL.
 
  1. Set `tinymce_live_demo_url` in `_config-local-override.yml`.
-    - This will override the `tinymce_live_demo_url` setting in `config.yml`.
+    - This will override the `tinymce_live_demo_url` setting in `antora.yml`.
     - This file is *not* intended to be checked in.
     - This option changes the URL for all live demos.
 
```

Comment:
The part below needs a minor update, as `live-demo.html` doesn't exist anymore. It should probably just say something like `when including the live demo.` instead I guess 🤷